app/models/question.rb

```diff
@@ -0,0 +1,30 @@
+class Question < ActiveRecord::Base
+  belongs_to :user
+  has_many :answers
+  has_many :question_tags
+  has_many :tags, through: :question_tags
+  has_many :votes, as: :votable
+  has_many :responses, as: :respondable
+
+  validates :title, :content, :user_id, presence: true
+  validates :title, uniqueness: true
+  validates :title, length: { maximum: 90 }
+  validates :content, length: { maximum: 10000 }
+
+  def set_posted_at
+    self.posted_at = self.created_at
+    self.save
+  end
+
+  def posted_at
+    super.strftime("on %m/%d/%Y at %I:%M%p")
+  end
+
+  def first(number)
+    if self.content.length > number
+      (self.content[0..number] + " <a href='/questions/#{self.id}'>...continue reading</a>").html_safe
```

Comment:
If you're constructing HTML, you should be doing it in the helpers, not the model.